LICENSE

```diff
@@ -200,3 +200,90 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+---
+
+This product bundles Sqlline (https://github.com/julianhyde/sqlline)
+which is licensed under the 3-clause BSD license
+
+Copyright (c) 2002,2003,2004,2005,2006,2007 Marc Prud'hommeaux
+Copyright (c) 2004-2010 The Eigenbase Project
+Copyright (c) 2013-2014 Julian Hyde
+All rights reserved.
+
+---
+
+This product bundles portions of AngularJS (https://angularjs.org/) which
```

Comment:
The text added here should be all that is needed for 4.8.0's source release for the trace ui stuff. The accompanying binary artifact just needs the same text copied into its LICENSE and NOTICE. While we can move this out for 4.8.0, I think I'd say we can decide what to do in a 4.9.0 (does this code get dropped, moved into some other repo, etc). We certainly can just remove it now if desired.